### PR TITLE
create index.md

### PR DESCRIPTION
**Pull request by sarthak0411**
https://github.com/freeCodeCamp/guides/commit/65907e45c7cf2826c52b273151884755ec44a7fcInstall and Configure Apache Server in Linux

### CODE REVIEW

**Comment by pawan-mehta** [comment]
This message was created automatically by mail delivery software.

A message that you sent could not be delivered to one or more of its
recipients. This is a temporary error. The following address(es) deferred:

  ossbackup2@gmail.com
    Domain oss-india.net has exceeded the max emails per hour (124/100 (124%)) allowed.  Message will be reattempted later

------- This is a copy of the message, including all the headers. ------
Received: from mail-oi0-f44.google.com ([209.85.218.44]:51440)
	by server14.hostingraja.in with esmtps (TLSv1.2:ECDHE-RSA-AES128-GCM-SHA256:128)
	(Exim 4.89)
	(envelope-from <pawan.ossindia+caf_=pawan=oss-india.net@gmail.com>)
	id 1e5Es1-001pYs-Kz
	for pawan@oss-india.net; Thu, 19 Oct 2017 23:14:51 +0530
Received: by mail-oi0-f44.google.com with SMTP id j126so16178868oib.8
        for <pawan@oss-india.net>; Thu, 19 Oct 2017 10:44:15 -0700 (PDT)
X-Google-DKIM-Signature: v=1; a=rsa-sha256; c=relaxed/relaxed;
        d=1e100.net; s=20161025;
        h=x-gm-message-state:delivered-to:dkim-signature:dkim-signature:date
         :from:reply-to:to:cc:message-id:subject:mime-version
         :content-transfer-encoding:precedence:list-id:list-archive:list-post
         :list-unsubscribe;
        bh=EcAd8CiKf3F0qTMQTBTsZ3BnbHNCA22BbqSxB44vkYs=;
        b=OjJOlK/F7RbHoo5m5FQr/WcaxFKmTsfI4E1zVUzOavvyGQzGSmu6yGL6ri+hjblzf/
         Aazl5HOMW0Hptlo2EWOQpJY+yE+J6Nys2ku9XWUjEXmiyNwTwxQqp8ImUagwgX+ms09P
         UrHZLQG11GDIFAog1/x4vNo/W83cFDzn8C4ytftX/zbcZFeEuwZWl1txVM9wzfr9e8gj
         gLYOphFCMBLAIKXvx2UCYQawpIOaQVPnzwndHHnCJAFsRlXybLxEG+XzCKS8Yc1E19R8
         mVjJr9s9+e4G0lPWrQzwwx4wn49YYMl0Tl7yrtyPooN+1JUXZAhei5TRO4nO395hlvL8
         xwLg==
X-Gm-Message-State: AMCzsaW7kMcpJ61NMBYZEQnthzqAYsK7EAf4nJPxS/9TKDR3tfM8KdqS
	yzMawPNNKFMxv18z9swG1IATeFNfSBaDxCE+lnGDgSE48P7pIPE=
X-Received: by 10.202.71.86 with SMTP id u83mr1303607oia.16.1508435033491;
        Thu, 19 Oct 2017 10:43:53 -0700 (PDT)
X-Forwarded-To: pawan@oss-india.net
X-Forwarded-For: pawan.ossindia@gmail.com pawan@oss-india.net
Delivered-To: pawan.ossindia@gmail.com
Received: by 10.168.68.11 with SMTP id r11csp4151859ofa;
        Thu, 19 Oct 2017 10:43:52 -0700 (PDT)
X-Google-Smtp-Source: ABhQp+TvM8SD/+/gyJoGUdJKd+NskzREVS38iihLcdjo2cyeiLkFuh+0ZmTPndn30quxsnMZhuu5
X-Received: by 10.107.88.4 with SMTP id m4mr2898935iob.287.1508435032809;
        Thu, 19 Oct 2017 10:43:52 -0700 (PDT)
ARC-Seal: i=1; a=rsa-sha256; t=1508435032; cv=none;
        d=google.com; s=arc-20160816;
        b=MJ/e6z6u1xfM1ZTfxj4NInZp/E3J+o7kua5PCppPxEKYBp327GEZ5L/6i3yjfZ3vmF
         ZrFZbCvKEp9lc+7OhxkMLO4007BbPmjJkU7AgfHjHBEysnL8Yif8KcjG+3kZ788A0AEa
         xMAnNF7r8U9sxMN7KrvP0ipOUu7cYMmhh384WZKbsXMB6zEZ9qz0faGHj2YT99W6yWvU
         mJH4HOmOt1/aMlsX2wHCSV/Mpm+yICVuz5WYGBz3jfDOSu2klNDwTsRsUzdAIPuZknsl
         b2DHPfbGfdrPhOOyES7QdWM501vfJBC73uPWi53jsgaXlLRILKjtMSI8sae6JA/PQ5G1
         BuKg==
ARC-Message-Signature: i=1; a=rsa-sha256; c=relaxed/relaxed; d=google.com; s=arc-20160816;
        h=list-unsubscribe:list-post:list-archive:list-id:precedence
         :content-transfer-encoding:mime-version:subject:message-id:cc:to
         :reply-to:from:date:dkim-signature:dkim-signature
         :arc-authentication-results;
        bh=EcAd8CiKf3F0qTMQTBTsZ3BnbHNCA22BbqSxB44vkYs=;
        b=pZyL1oUSixgGV8vMAWZB85wahbNAZFMyl5H9Lvccau68MCeCwvwrlaesDABFGEfPaB
         aiSNYWiQh8qJmKSLt/y0LS7hLGaDLQ+GxISYT8obkhkbnxUegMxBVpvaobt2iqmOkVPN
         ZZwEN9H2EM+OoA0DygmiBfElyWXw17rcqCuFmEpmQXO6j2mLtKjR0X+mbU724EhiGGk9
         2mSuE15x3VUvhC/+2FV53FD28C+U1IEX5d3NiAcyBzD9cuYPovS1l2NGXw+WU0rG2cOI
         Y0FW9Vo4o5kW6ubnjbflBtN4RxvN1b7SJHqGEYoN0rRwvOI+eNTx8KgaZEm6G2iQWh7J
         NHjw==
ARC-Authentication-Results: i=1; mx.google.com;
       dkim=pass header.i=@github.com header.s=s20150108 header.b=nWnhSyIx;
       dkim=pass header.i=@sendgrid.info header.s=smtpapi header.b=B7lx6kWB;
       spf=pass (google.com: domain of bounces+848413-3702-pawan.ossindia=gmail.com@sgmail.github.com designates 167.89.101.198 as permitted sender) smtp.mailfrom=bounces+848413-3702-pawan.ossindia=gmail.com@sgmail.github.com;
       dmarc=pass (p=NONE sp=NONE dis=NONE) header.from=github.com
Received: from o7.sgmail.github.com (o7.sgmail.github.com. [167.89.101.198])
        by mx.google.com with ESMTPS id j65si6477787ioo.113.2017.10.19.10.43.52
        for <pawan.ossindia@gmail.com>
        (version=TLS1_2 cipher=ECDHE-RSA-AES128-GCM-SHA256 bits=128/128);
        Thu, 19 Oct 2017 10:43:52 -0700 (PDT)
Received-SPF: pass (google.com: domain of bounces+848413-3702-pawan.ossindia=gmail.com@sgmail.github.com designates 167.89.101.198 as permitted sender) client-ip=167.89.101.198;
Authentication-Results: mx.google.com;
       dkim=pass header.i=@github.com header.s=s20150108 header.b=nWnhSyIx;
       dkim=pass header.i=@sendgrid.info header.s=smtpapi header.b=B7lx6kWB;
       spf=pass (google.com: domain of bounces+848413-3702-pawan.ossindia=gmail.com@sgmail.github.com designates 167.89.101.198 as permitted sender) smtp.mailfrom=bounces+848413-3702-pawan.ossindia=gmail.com@sgmail.github.com;
       dmarc=pass (p=NONE sp=NONE dis=NONE) header.from=github.com
DKIM-Signature: v=1; a=rsa-sha1; c=relaxed/relaxed; d=github.com; 
	h=from:reply-to:to:cc:subject:mime-version:content-type:content-transfer-encoding:list-id:list-archive:list-post:list-unsubscribe; 
	s=s20150108; bh=xIjIL3yXF/VQHMr4DzASjOUO2bM=; b=nWnhSyIxXGyZ0g8H
	cnSRSWru9tb2Hta/Ad881C/uAjTWdcJJUal+NBVL6ccrUscKCzQiuCTs5oG/56vy
	HfDjT8Ypubi+iRKFEWR5oMgrrsEmM3J4VvhpFhycbptxRHBILzqmV/ohRaMwn3Lo
	FoYphEqdLiPrs6MBV+AG9/JD69Q=
DKIM-Signature: v=1; a=rsa-sha1; c=relaxed/relaxed; d=sendgrid.info; 
	h=from:reply-to:to:cc:subject:mime-version:content-type:content-transfer-encoding:list-id:list-archive:list-post:list-unsubscribe:x-feedback-id; 
	s=smtpapi; bh=xIjIL3yXF/VQHMr4DzASjOUO2bM=; b=B7lx6kWBpRU0k+Oj29
	wN8N5ArNIfPhvb61Wu6CVowJccy5x5RlGJrrzJAKEyd6HL2FpXntSdB3KhhniwXI
	UXwGLNSUA3DcdWzpX5F0sYmoCFgWO1PloGhzASjwLuJbRjJYqiMItBpZCXDALuhf
	t/5lbSdxSlXOQzveFkR3Hzc14=
Received: by filter1090p1mdw1.sendgrid.net with SMTP id filter1090p1mdw1-7420-59E8E458-2B
        2017-10-19 17:43:52.745257681 +0000 UTC
Received: from github-smtp2b-ext-cp1-prd.iad.github.net (github-smtp2b-ext-cp1-prd.iad.github.net [192.30.253.17])
	by ismtpd0006p1iad1.sendgrid.net (SG) with ESMTP id ZittVQjhTwKSQRV3QgZGUg
	for <pawan.ossindia@gmail.com>; Thu, 19 Oct 2017 17:43:52.502 +0000 (UTC)
Date: Thu, 19 Oct 2017 17:43:52 +0000 (UTC)

File: src/pages/linux/install-and-configure-apache-server-in-linux/index.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+---
+title: Install and Configure Apache Server in Linux
+---
+
+<div class="para">Apache HTTP Server, colloquially called Apache, is free and open-source
```
Comment (Bouncey) [inline src/pages/linux/install-and-configure-apache-server-in-linux/index.md]: You do not need to use any kind of markup here, this is all done behind the scenes at build time

File: src/pages/linux/install-and-configure-apache-server-in-linux/index.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
+---
+title: Install and Configure Apache Server in Linux
+---
+
+<div class="para">Apache HTTP Server, colloquially called Apache, is free and open-source 
+cross-platform web server software.
+Apache is developed and maintained by an 
+open community of developers under the auspices of the Apache Software Foundation.</div>
+
+# Install and Configure Apache Server in Linux
```
Comment (Bouncey) [inline src/pages/linux/install-and-configure-apache-server-in-linux/index.md]: ~# Install and Configure Apache Server in Linux~

`## Install and Configure Apache Server in Linux `

File: src/pages/linux/install-and-configure-apache-server-in-linux/index.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+Step1: Install Apache Server<br>
```
Comment (Bouncey) [inline src/pages/linux/install-and-configure-apache-server-in-linux/index.md]: ~Step1~

Step 1

File: src/pages/linux/install-and-configure-apache-server-in-linux/index.md [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+```#yum install httpd```
```
Comment (Bouncey) [inline src/pages/linux/install-and-configure-apache-server-in-linux/index.md]: A simpler we of writing this in md would be
```sh
`yum install httpd`
```

We have also found that indication a bash/shell command with a `#` or `$` can be confusing to newer developers. Please remove them

File: src/pages/linux/install-and-configure-apache-server-in-linux/index.md [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+
+Step2: Starting Apache Server<br>
```
Comment (Bouncey) [inline src/pages/linux/install-and-configure-apache-server-in-linux/index.md]: As above

File: src/pages/linux/install-and-configure-apache-server-in-linux/index.md [hunk 6]
```diff
@@ -0,0 +1,19 @@
...
+```#systemctl start httpd.service```
```
Comment (Bouncey) [inline src/pages/linux/install-and-configure-apache-server-in-linux/index.md]: As above

File: src/pages/linux/install-and-configure-apache-server-in-linux/index.md [hunk 7]
```diff
@@ -0,0 +1,19 @@
...
+
+Step3: Automatically Start Apache Server when Operating System start.<br>
```
Comment (Bouncey) [inline src/pages/linux/install-and-configure-apache-server-in-linux/index.md]: As above

File: src/pages/linux/install-and-configure-apache-server-in-linux/index.md [hunk 8]
```diff
@@ -0,0 +1,19 @@
...
+```#systemctl enable httpd.service```
```
Comment (Bouncey) [inline src/pages/linux/install-and-configure-apache-server-in-linux/index.md]: As above

**Pull request merged**